external-crates/move/crates/move-analyzer/tests/colon_colon_completion.snap

```diff
@@ -23,6 +23,7 @@ Unit '0x1'
 Unit '0xA'
 Unit '0xCAFE'
 Unit 'Completion'
+Unit 'SamePkgAddr'
```

Comment:
This changed due to adding dependency on a package with the same address. Printing of names of such packages was actually correctly handled for auto-completions (but not for auto-imports...) but I figured we should be testing for it anyway just in case